Block/CommentThreadAsyncBlockService.php

```diff
@@ -28,10 +28,8 @@ class CommentThreadAsyncBlockService extends BaseBlockService
     /**
      * {@inheritdoc}
      */
-    public function setDefaultSettings(OptionsResolverInterface $resolver)
+    public function configureSettings(OptionsResolver $resolver)
     {
-        parent::setDefaultSettings($resolver);
```

Comment:
Not needed, the parent is BaseBlock which is an empty method (actually in AbstractBlock)